docs/guides/push-work-pools.md

```diff
@@ -32,12 +34,35 @@ In this guide you will:
     Once the Service account is created, navigate to its *Keys* page to add an API key. Create a JSON type key, download it, and store it somewhere safe for use in the next section.
 
 === "AWS ECS"
+    
     To push work to ECS, AWS credentials are required.
 
     Create a user and attach the *AmazonECS_FullAccess* permissions.
 
     From that user's page create credentials and store them somewhere safe for use in the next section.
 
+=== "Azure Container Instance"
+    
+    To push work to Azure, an Azure subscription, resource worker and tenant secret are required. 
+
+    ##### Create Subscription and Resource Worker
+
+    1. In the Azure portal, create a subscription.
+    2. Create a resource group within your subscription.
+
+    ### Create App Registration
+
+    1. In the Azure portal, create an app registration.
+    2. In the app registration, create a client secret. Copy the value and store it somewhere safe.
+    
+    ### Add App Registration to Subscription
+
+    1. Navigate to the resource group you created earlier.
+    2. Click on "Access control (IAM)" and then "Role assignments".
+    3. Search for the app registration and select it. Give it a role that has sufficient privileges to create, run, and delete ACI container groups.
```

Comment:
They can either add contributor to a specific resource group (that they created) to limit the scope, or add a custom role / permissions with explicit `Microsoft.ContainerInstance/containerGroups/*` - still probably good to scope that to the resource group. 

I'm not sure how detailed we need to be here though for users - RBAC and permissions should fall to them?

Reply:
Yeah being too prescriptive on RBAC and permissions get difficult. I think I'll leave as is and we can update with more info if we get questions about it.